js/tidex2.js

```diff
@@ -0,0 +1,960 @@
+                'setPositionMode': false,
+                'withdraw': true,
+            },
+            'urls': {
+                'logo': 'https://user-images.githubusercontent.com/1294454/30781780-03149dc4-a12e-11e7-82bb-313b269d24d4.jpg',
+                'api': {
+                    'web': 'https://gate.tidex.com/api',
+                    'public': 'https://api.tidex.com/api/v1/public',
+                    'private': 'https://api.tidex.com/api/v1',
+                },
+                'www': 'https://tidex.com',
+                'doc': 'https://gitlab.com/tidex/api/-/blob/main/tidex_doc.md',
+                'referral': 'https://tidex.com/exchange/?ref=57f5638d9cd7',
+                'fees': [
+                    'https://tidex.com/fee-schedule',
+                ],
+            },
+            'timeframes': {
+                '15s': '15',
+                '1m': '60',
+                '5m': '300',
+                '15m': '900',
+                '1h': '3600',
+                '4h': '14400',
+                '1d': '86400',
+                '3d': '259200',
+                '1w': '604800',
+            },
+            'api': {
+                'web': {
+                    'get': [
+                        'currency',
+                    ],
+                },
+                'public': {
+                    'get': [
+                        'markets',
+                        'tickers',
+                        'ticker',
+                        'book',
+                        'history/result',
+                        'symbols',
+                        'depth/result',
+                        'kline',
+                    ],
+                },
+                'private': {
+                    'post': [
+                        'account/balances',
+                        'account/balance',
+                    ],
+                },
+            },
+            'fees': {
+                'trading': {
+                    'feeSide': 'get',
+                    'tierBased': false,
+                    'percentage': true,
+                    'taker': this.parseNumber ('0.002'),
+                    'maker': this.parseNumber ('0.002'),
+                },
+            },
+            'commonCurrencies': {
+                'DSH': 'DASH',
+                'EMGO': 'MGO',
+                'MGO': 'WMGO',
+            },
+            'requiredCredentials': {
+                'apiKey': true,
+                'secret': true,
+            },
+            'exceptions': {
+                'exact': {
+                },
+                'broad': {
+                    'Api key header is missing!': AuthenticationError, // {"code":0,"success":false,"message":"Api key header is missing!","result":[]}
+                },
+            },
+            'options': {
+                'fetchTickersMaxLength': 1000,
+            },
+            'orders': {}, // orders cache / emulation
+        });
+    }
+
+    async fetchCurrencies (params = {}) {
```

Comment:
I think we can probably remove the `fetchCurrencies ( )` method and the `web` endpoint  from the `api` property - currencies can be populated in other ways too